txpipe/twopoint_null_tests.py

```diff
@@ -766,3 +767,272 @@ def write_output(self, source_list, lens_list, meta, results):
 
         # Finally, save the output to Sacc file
         S.save_fits(self.get_output("aperture_mass_data"), overwrite=True)
+
+
+
+class TXpureB(PipelineStage):
+    '''
+    Make shear B-mode measurements
```

Comment:
Added:

>    
    Make shear B-mode measurements.
    This stage computes xip and xim in very narrow angular bins and transforms them into 
    Fourier B-modes using narrow window functions. This allows us to compute B-modes without
    being affected by (a) the mask (which is an issue if we use https://arxiv.org/abs/astro-ph/0511629,
    which is implemented in NaMaster, since masks for LSS is more complicated than CMB) and
    (b) noise bias (since the measurements we make here are real-space measurements).